contentful_management/resource.py

```diff
@@ -274,10 +274,8 @@ def create_attributes(klass, attributes, previous_object=None):
 
         result = super(MetadataResource, klass).create_attributes(attributes, previous_object)
 
-        if '_metadata' not in attributes:
-            result['metadata'] = {}
-        else:
-            result['metadata'] = attributes['_metadata']
+        if '_metadata' in attributes:
+            result['metadata'] = attributes.pop('_metadata')
```

Comment:
calling .pop on a dictionary will remove and return this. is that desired?

Reply:
yes as `_metadata` is naming convention used by Python SDK and is irrelevant when making API request.